ocfweb/account/templates/account/register/index.html

```diff
@@ -139,6 +138,6 @@ <h4>OCF &hearts; Python, Linux, Open Source</h4>
     </div>
 {% endblock %}
 {% block inline_js %}
-        <script src="/static/js/pages/account/quick_username_validation.js"></script>
-        <script src="/static/js/pages/account/recommend_username.js"></script>
+        <script src="{% static 'js/pages/account/quick_username_validation.js' %}"></script>
+        <script src="{% static 'js/pages/account/recommend_username.js' %}"></script>
```

Comment:
i wonder if we should combine these two scripts into one

the size isn't an issue, but multiple requests can suck on bad connections (e.g. phones, where people often make new accounts)

Reply:
Sure, should I just throw under a new register.js?

Reply:
sounds reasonable to me!